requirements/postgres.txt

```diff
@@ -1,3 +1,5 @@
 -r base.txt
 
 psycopg2==2.9.3
+
+sentry-sdk==1.5.8
```

Comment:
?

Reply:
If production environment uses "PostgreSQL" it can't use "production.txt" but has to use "postgres.txt".
We need to clean this mess up, until than ... it's a dup.

Reply:
I see.